build.sbt

```diff
@@ -28,7 +28,8 @@ lazy val parent = (project in file("."))
   .settings(
     name := "login-gateway",
     javacOptions ++= Seq("-source", "1.8", "-target", "1.8", "-Xlint"),
-    publish / skip := true
+    publish / skip := true,
+    javaOptions ++= Seq("-Dspring.jmx.enabled=true", "-Dmanagement.endpoints.jmx.exposure.include=health")
```

Comment:
Btw. Have you tried this? E.g. using JConsole or with another tool?

If you were able to provide short info regarding the JMX connection in the README (a few explanatory sentences and perhaps a snippet on how to connect), this would help the potential starting users immensely.

Reply:
Will add to ReadMe